sdk/ai/Azure.AI.Projects/tests/Samples/Agent/Sample_Agent_Additional_messages.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.Projects.Tests;
+public partial class Sample_Agent_Multiple_Messages : SamplesBase<AIProjectsTestEnvironment>
+{
+    [Test]
+    public async Task CreateAdditionalMessageExample()
+    {
+#if SNIPPET
+        var connectionString = Environment.GetEnvironmentVariable("PROJECT_CONNECTION_STRING");
+#else
```

Comment:
This condition is unnecessary here since the code is not part of any snippet.

Reply:
Removed